charts/de-elasticsearch/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v2
+name: de-elasticsearch
+description: A Helm chart for elastic search in data explorer
+type: application
+version: 0.1.0
+maintainers:
+  - email: tjiang@broadinstitute.org
+    name: tjiang
+dependencies:
+  - name: elasticsearch
+    version: 7.10.2
```

Comment:
The current release is 7.12.1, is there a reason to not use the latest?

Reply:
@fboulnois mentioned this is the newest version with open source, due to some licensing reason we have to rely on the open source version

Reply:
Yes. 7.10.2 is the last OSS-licensed version. Newer versions are no longer OSS.